changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## NEXT
+
+- Made `cai_causal_graph.causal_graph.CausalGraph.copy` more general to preserve the class type. Removed `.copy` from
+  the `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` class.
+- Extended equality method for the `cai_causal_graph.causal_graph.CausalGraph`. Now it has a `deep` parameter that
+  forces checks on all class attributes for nodes and edges. To call you must do `graph_1.__eq__(graph_2, deep=True)`
```

Comment:
"forces deep equality checks on all nodes and edges". We should revise changelog in 0.2.3 to not actually say all class attributes. Would say if deep, additional class attributes are also checked.